src/topology/continuous_function/stone_weierstrass.lean

```diff
@@ -112,7 +112,7 @@ begin
   let M := ∥f∥,
   let f' := attach_bound (f : C(X, ℝ)),
   let abs : C(set.Icc (-∥f∥) (∥f∥), ℝ) :=
-  { to_fun := λ x : set.Icc (-∥f∥) (∥f∥), _root_.abs (x : ℝ) },
+  { to_fun := λ x : set.Icc (-∥f∥) (∥f∥), has_abs.abs (x : ℝ) },
```

Comment:
Similarly here, why not use the notatation?

Reply:
I was probably saving it for the other PR https://github.com/leanprover-community/mathlib/pull/8891 . I've updated this PR to use the notation now.